Objects/object.c

```diff
@@ -3405,3 +3387,16 @@ _PyObject_VisitType(PyObject *op, visitproc visit, void *arg)
     Py_VISIT(tp);
     return 0;
 }
+
+// Implementations for the stable ABI
+// Keep these at the end.
+#undef Py_TYPE
+#undef Py_REFCNT
+#undef Py_SIZE
+#undef Py_IS_TYPE
+#undef Py_SET_SIZE
+PyTypeObject* Py_TYPE(PyObject *ob) { return _Py_TYPE_impl(ob); }
+Py_ssize_t Py_REFCNT(PyObject *ob) { return _Py_REFCNT(ob); }
+Py_ssize_t Py_SIZE(PyObject *o) { return _Py_SIZE_impl(o); }
+int Py_IS_TYPE(PyObject *o, PyTypeObject *t) { return _Py_IS_TYPE_impl(o, t); }
+void Py_SET_SIZE(PyVarObject *o, Py_ssize_t s) { _Py_SET_SIZE_impl(o, s); }
```

Comment:
Why not using the regular PEP 7 formatting?

```suggestion
PyTypeObject*
Py_TYPE(PyObject *ob)
{
    return _Py_TYPE_impl(ob);
}

Py_ssize_t
Py_REFCNT(PyObject *ob)
{
    return _Py_REFCNT(ob);
}

Py_ssize_t
Py_SIZE(PyObject *o)
{
    return _Py_SIZE_impl(o);
}

int
Py_IS_TYPE(PyObject *o, PyTypeObject *t)
{
    return _Py_IS_TYPE_impl(o, t);
}

void
Py_SET_SIZE(PyVarObject *o, Py_ssize_t s)
{
    _Py_SET_SIZE_impl(o, s);
}
```

Reply:
I think it's easier to read in the compact form. If every line has the same structure it's easier to scan that the names match.
Plus, the comment is more visible as you don't need to scroll up to see it.

Reply:
Ok, as you want. I don't have a strong preference.